### PR TITLE
add position to generator, mcs and load

### PR DESCRIPTION
**Pull request by bendaamerahm**
Signed-off-by: bendaamerahm <ahmed.bendaamer@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/service/NetworkStoreListener.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public static NetworkStoreListener create(Network network, UUID networkUuid, UUI
                                               EquipmentInfosService equipmentInfosService,
                                               boolean isBuild, boolean isApplyModifications) {
         var listener = new NetworkStoreListener(network, networkUuid, groupUuid, modificationRepository, equipmentInfosService,
-            isBuild, isApplyModifications);
+                isBuild, isApplyModifications);
```
Comment (laurgarn) [inline src/main/java/org/gridsuite/modification/server/service/NetworkStoreListener.java:76]: All changes in this source seem to be indentation changes.

Comment (bendaamerahm) [reply src/main/java/org/gridsuite/modification/server/service/NetworkStoreListener.java]: my bad

**Review by geofjamg — APPROVED** [review]

File: src/main/java/org/gridsuite/modification/server/entities/equipment/creation/GeneratorCreationEntity.java [hunk 2]
```diff
@@ -102,8 +105,10 @@ public GeneratorCreationEntity(String equipmentId, String equipmentName, EnergyS
                                    double minActivePower, double maxActivePower, Double ratedNominalPower, double activePowerSetpoint,
                                    Double reactivePowerSetpoint, boolean voltageRegulationOn, Double voltageSetpoint, Double marginalCost, Double minQ,
                                    Double maxQ, boolean participate, Float droop, Double transientReactance, Double stepUpTransformerReactance,
+
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/modification/server/entities/equipment/creation/GeneratorCreationEntity.java:108]: no empty line

File: src/main/java/org/gridsuite/modification/server/entities/equipment/creation/GeneratorCreationEntity.java [hunk 3]
```diff
@@ -102,8 +105,10 @@ public GeneratorCreationEntity(String equipmentId, String equipmentName, EnergyS
...
                                    List<ReactiveCapabilityCurveCreationEmbeddable> reactiveCapabilityCurvePoints, String regulatingTerminalId, String regulatingTerminalType, String regulatingTerminalVlId,
-                                   Double qPercent, Boolean reactiveCapabilityCurve, String connectionName, ConnectablePosition.Direction connectionDirection) {
+                                   Double qPercent, Boolean reactiveCapabilityCurve, String connectionName, ConnectablePosition.Direction connectionDirection, Integer connectionPosition) {
+
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/modification/server/entities/equipment/creation/GeneratorCreationEntity.java:111]: same

**Review by etiennehomer — CHANGES_REQUESTED** [review]

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**